wiki/Disambiguation/Offset/en.md

```diff
@@ -0,0 +1,16 @@
+# Offset (disambiguation)
+
+**Offset** can have multiple meanings:
+
+## Timing settings
+
+- An [uninherited ("red") offset](/wiki/Beatmapping/Offset), which denotes either the song's first downbeat, or a start of a new timing section.
+- [Online offset](/wiki/Beatmaps/Online_offset), a timing change applied by [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) to ranked beatmaps with incorrect uninherited offset in order to adjust it.
+- [Local ("offline") offset](/wiki/Glossary#local-song-offset), a per-beatmap setting that shifts the objects in time during gameplay.
+- [Universal offset](/wiki/Universal_offset), the osu! client setting that globally affects the timing of beatmaps.
+
+## Beatmap settings
+
+- Countdown offset, a setting which affects the appearance of [countdown](/wiki/Beatmapping/Countdown) before the first hit object.
+- Audio offset, or [audio lead-in](/wiki/Glossary/Lead-in).
+- Video offset, a setting that shows how soon a background video should play relatively to the audio track, if a beatmap has one.
```

Comment:
```suggestion
- Video offset, a setting that shows how soon a background video should play relatively to the audio track.
```
Otherwise it sounds a beatmap *can* have an audio track ¯\_(ツ)_/¯